lib/python/setup.py

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env python3
+import os
+from sys import platform
+from setuptools import setup, Extension
+
+##Determine what system we are building on to determine what type of shared object has been built and needs copyingthis differs between OS's (e.g. libdidkit.so vs libdidkit.dylib vs libdidkit.dll)
+didpath = "didkit"
+so_filename = "libdidkit"
+if platform == "linux" or platform == "linux2":
+    LIBDIDKIT_SHARE_OBJ = os.path.join(didpath, '%s.so'%(so_filename))
+elif platform == "darwin":
+    LIBDIDKIT_SHARE_OBJ = os.path.join(didpath, '%s.dylib'%(so_filename))
+else:
+    LIBDIDKIT_SHARE_OBJ = os.path.join(didpath, '%s.dll'%(so_filename))
```

Comment:
For Windows the name of the shared object is `didkit.dll`, a correction is needed at [this](https://github.com/spruceid/didkit/blob/main/lib/python/didkit/__init__.py#L16) file also. Also it would be better to use [platform](https://docs.python.org/3/library/platform.html#platform.system) since it has only 4 possible values, we could treat them all an throw an error if empty string.

Reply:
@w4ll3 you should have permission to add commits to this branch, could you perhaps move forward with your recommended changes? Would be great to get this merged before too many other code changes come in. Thanks.

Reply:
Suggestions and fixes incorporated. The system type of "Java" didn't have a clear shared object so I left it to be caught by the final else clause but let me know if Java support is something DIDKit has.

Reply:
Thanks! Sorry, didn't mean to rush you on this--should have confirmed if it was okay with you. We are still figuring out contributor policies & timelines, so appreciate the understanding.

Reply:
Oh not at all, I had this ready to go I just hadn't merged it and got distracted onto other things. Thanks for the reminder :)